lib/rmt/cli/products.rb

```diff
@@ -36,41 +36,80 @@ def list
-  long_desc <<-REPOS
-#{_('Enable mirroring of product repositories by a list of product IDs or product strings.')}
+  long_desc <<~REPOS
+    #{_('Enable mirroring of product repositories by a list of product IDs or product strings.')}
 
-#{_('Examples')}:
+    #{_('Examples')}:
 
-$ rmt-cli products enable SLES/15
+    $ rmt-cli products enable SLES/15
 
-$ rmt-cli products enable 1575
+    $ rmt-cli products enable 1575
 
-$ rmt-cli products enable SLES/15/x86_64 1743
+    $ rmt-cli products enable SLES/15/x86_64 1743
 
-$ rmt-cli products enable --all-modules SLES/15
-REPOS
+    $ rmt-cli products enable --all-modules SLES/15
+  REPOS
   def enable(*targets)
     change_products(targets, true, options[:all_modules])
   end
 
   desc 'disable TARGETS', _('Disable mirroring of product repositories by a list of product IDs or product strings.')
-  long_desc <<-REPOS
-#{_('Disable mirroring of product repositories by a list of product IDs or product strings.')}
+  long_desc <<~REPOS
+    #{_('Disable mirroring of product repositories by a list of product IDs or product strings.')}
 
-#{_('Examples')}:
+    #{_('Examples')}:
 
-$ rmt-cli products disable SLES/15
+    $ rmt-cli products disable SLES/15
 
-$ rmt-cli products disable 1575
+    $ rmt-cli products disable 1575
 
-$ rmt-cli products disable SLES/15/x86_64 1743
-REPOS
+    $ rmt-cli products disable SLES/15/x86_64 1743
+  REPOS
   def disable(*targets)
     change_products(targets, false, false)
   end
 
+  desc 'show TARGET', _('Displays product with all its repositories and their attributes.')
+  long_desc <<~SHOW
+    #{_('Displays product with all its repositories and their attributes.')}
+
+    #{_('Examples')}:
+
+    $ rmt-cli products show SLES/15
+
+    $ rmt-cli products show 1575
+  SHOW
+  def show(target)
+    show_product(target)
+  end
+
   protected
 
+  def show_product(target)
+    product = find_products(target).first
+
+    raise ProductNotFoundException.new(_('No product found for target %{target}.') % { target: target }) if product.blank?
+
+    puts _('Product: %{name} (id: %{id})') % { name: product.friendly_name, id: product.id }
+    puts _('Description: %{description}') % { description: product.description }
+    show_product_repos(product)
+  rescue ProductNotFoundException => e
+    puts e.message
+  end
+
+  def show_product_repos(product)
+    repos = product.repositories
+      .pluck(:name, :scc_id, :enabled, :mirroring_enabled, :last_mirrored_at)
+    raise RMT::CLI::Error.new(_('Repositories are not available for this product.')) if repos.blank?
+    puts _('Repositories:')
+    repos.each do |repo|
+      repo[2] = repo[2] ? 'mandatory' : 'non-mandatory'
+      repo[3] = repo[3] ? 'enabled' : 'not enabled'
+      repo[4] = repo[4].present? ? "mirrored at #{repo[4].strftime('%Y-%m-%d %H:%M:%S %Z')}" : 'not mirrored'
+      puts _("* #{repo[0]} (id: #{repo[1]}) (#{repo[2]}, #{repo[3]}, #{repo[4]})")
```

Comment:
This won't work correctly with i18n -- an interpolated string is being passed to to `_()` in runtime (.po files won't be generated correctly).

Reply:
@ikapelyukhin Thanks! Would you recommend to use a template?

Reply:
Yes, like it's done here for example: https://github.com/SUSE/rmt/blob/f48968e6cca4bba22b76806d83b3a6d7e7ca3f5c/lib/rmt/cli/products.rb#L93-L94 

Use `_()` on regular strings, e.g.: 
```
repo[2] ? _('mandatory') : _('non-mandatory')
```
And then use internationalized strings in the template.

Reply:
@ikapelyukhin Great, thank you!